src/repocontributors/repocon.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, area, reString):
         self.re = re.compile(self.reString)
 
 areaREs = [
-        areaRE('documentation', r'doc/'),
+        areaRE('documentation', r'docs/'),
```

Comment:
I don't know if this is useful in anything other than chisel2. If you aren't actually using it, I'd undo this edit just so it's obvious the file is way out of date.